uv.lock

```diff
@@ -0,0 +1,1053 @@
+version = 1
```

Comment:
maybe consider using https://peps.python.org/pep-0751/ see https://github.com/astral-sh/uv/issues/12584

Reply:
No objections there, I'll get the lockfile converted.

Reply:
Hmm, actually it seems like uv doesn't consume `pylock.toml` files in normal operation, only under `uv pip` subcommands - see [docs](https://docs.astral.sh/uv/concepts/projects/layout/#pylocktoml)